node/src/accountant/scanners.rs

```diff
@@ -966,7 +1023,7 @@ mod tests {
         let (_, _, all_non_pending_payables) = make_payables(now, &PaymentThresholds::default());
         let payable_dao =
             PayableDaoMock::new().non_pending_payables_result(all_non_pending_payables);
-        let mut subject = PayableScanner::default().payable_dao(payable_dao);
+        let mut subject = PayableScannerBuilder::default().payable_dao(payable_dao);
```

Comment:
Yeah...the bad taste from this still is in my mouth. Hoping you will discard the obfuscating new types. Then I will smile again :slightly_smiling_face: .